README.md

```diff
@@ -1,15 +1,107 @@
-# Eppo SDK for Python
+# Eppo Python SDK
 
-## Getting Started
+[![Test and lint SDK](https://github.com/Eppo-exp/python-sdk/actions/workflows/test-and-lint-sdk.yml/badge.svg)](https://github.com/Eppo-exp/python-sdk/actions/workflows/test-and-lint-sdk.yml)
 
-Refer to our [SDK documentation](https://docs.geteppo.com/feature-flags/sdks/python) for how to install and use the SDK.
+[Eppo](https://www.geteppo.com/) is a modular flagging and experimentation analysis tool. Eppo's Python SDK is built to make assignments in multi-user server side contexts. Before proceeding you'll need an Eppo account.
 
-## Supported Python Versions
+## Features
 
-This version of the SDK is compatible with Python 3.6 and above.
+- Feature gates
+- Kill switches
+- Progressive rollouts
+- A/B/n experiments
+- Mutually exclusive experiments (Layers)
+- Global holdouts
+- Dynamic configuration
 
-## Development
+## Installation
 
-### Running tests
+```javascript
+pip install eppo-server-sdk
+```
 
-`make test`
+## Quick start
+
+Begin by initializing a singleton instance of Eppo's client. Once initialized, the client can be used to make assignments anywhere in your app.
+
+#### Initialize once
+
+```python
+import eppo_client
+from eppo_client.config import Config, AssignmentLogger
+
+client_config = Config(api_key="<SDK-KEY-FROM-DASHBOARD>")
+eppo_client.init(client_config)
+```
+
+
+#### Assign anywhere
+
+```python
+import eppo_client
+
+client = eppo_client.get_instance()
+user = get_current_user()
+
+variation = eppoClient.get_boolean_assignment('show-new-feature', user.id, {
+    'country': user.country,
+    'device': user.device,
+}, False)
+```
+
+## Assignment functions
+
+Every Eppo flag has a return type that is set once on creation in the dashboard. Once a flag is created, assignments in code should be made using the corresponding typed function:
+
+```javascript
+get_bool_assignment(...)
+get_numeric_assignment(...)
+get_integer_assignment(...)
+get_string_assignment(...)
+get_json_assignment(...)
+```
+
+Each function has the same signature, but returns the type in the function name. For booleans use `get_boolean_assignment`, which has the following signature:
+
+```python
+get_boolean_assignment(
+    flag_key: str,
+    subject_key: str,
+    subject_attributes: Dict[str, Any],
+    default_value: bool
+) -> bool:
```

Comment:
Is this the pythonic way to show a signature?  Seems to highlight `->` as an error

Reply:
Yes it's correct